src/Gui/Quarter/QuarterWidget.cpp

```diff
@@ -1053,14 +1047,6 @@ QuarterWidget::redraw(void)
   this->viewport()->update();
 #else
 
-// #if QT_VERSION >= 0x050500 && QT_VERSION < 0x050600
-#if 1
-  // With Qt 5.5.x there is a major performance problem
-  this->viewport()->update();
-#else
-  this->viewport()->repaint();
-#endif
-
 #endif
```

Comment:
Do we really need to save the `#if`-`#else`-`#endif` construct?

Reply:
I don't think so, I'll remove it.